src/Helper/DependencyInjection.php

```diff
@@ -1,26 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Ssch\TYPO3Rector\Helper;
-
-use BadMethodCallException;
-use function Symfony\Component\DependencyInjection\Loader\Configurator\ref;
-use Symfony\Component\DependencyInjection\Loader\Configurator\ReferenceConfigurator;
-use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
-
-final class DependencyInjection
-{
-    public static function service(string $serviceId): ReferenceConfigurator
-    {
-        if (function_exists('Symfony\Component\DependencyInjection\Loader\Configurator\service')) {
-            return service($serviceId);
-        }
-
-        if (function_exists('Symfony\Component\DependencyInjection\Loader\Configurator\ref')) {
-            return ref($serviceId);
-        }
-
-        throw new BadMethodCallException('Cannot resolve one of the symfony di functions ref or service');
-    }
-}
```

Comment:
This is never used

Reply:
I know. This was a little hack, because in some of the projects of TYPO3 old versions auf symfony/dependency-injections is used. Therefore this function service does not exists but ref. I am looking forward to have a scoped version of typo3-rector. Then this problem is gone.